client/src/components/LoginForm.js

```diff
@@ -1,11 +1,13 @@
+// @flow
+
 import React from 'react';
 import {
     Input,
     Button
 } from 'reactstrap';
 import './LoginForm.css';
 
-const LoginForm = ({ onSubmit }) => {
+const LoginForm = ({ onSubmit }: any) => {
```

Comment:
@rahil051 the whole point of adding typing is not to use any - so we know whats expected

Reply:
@mashhoodr function type didn't worked, therefore added any for now. Will removed it in the next commit